justfile

```diff
@@ -1,29 +1,32 @@
 alias build := youki-release
 alias youki := youki-dev
 
-ROOT := `git rev-parse --show-toplevel`
+KIND_CLUSTER_NAME := 'youki'
 
 # build
 
 # build all binaries
 build-all: youki-release rust-oci-tests-bin runtimetest
 
 # build youki in dev mode
+[no-cd]
```

Comment:
May I ask what are we trying to achieve with the `[no-cd]` attribute? I don't think we need this attribute here because this is a justfile at the project root.

Reply:
I forgot to delete them.